Source/DafnyCore/DafnyOptions.cs

```diff
@@ -426,6 +427,14 @@ public enum IncludesModes {
           WarnShadowing = true;
           return true;
 
+        case "optimizeInvisibleDatatypeWrappers": {
```

Comment:
If you want this option to be available in the new CLI UI (and you do), then you have to add a value of `IOptionSpec` to `ICommandSpec.ExecutionOptions`. You can add the same value to `DafnyOptions.AvailableNewStyleOptions` to also make it available in the old UI.

Reply:
Indeed, I do. I changed it. Well, I don't understand if the `Description` should talk about arguments `0` and `1` or arguments `false` and `true`.

Reply:
> Well, I don't understand if the Description should talk about arguments 0 and 1 or arguments false and true

Yes that's a problem. I'm adding the option of having two descriptions here: https://github.com/dafny-lang/dafny/pull/3154